test/cpp/test_ir.cpp

```diff
@@ -94,5 +95,91 @@ TEST(IrTest, TestScopePusherWithDebugging) {
   FLAGS_torch_lazy_ir_debug = restore_FLAGS_torch_lazy_ir_debug;
 }
 
+TEST(IrTest, TestSizeNode) {
+  torch::lazy::NodePtr scalar_node =
+      ScalarOp(1.0, xla::ShapeUtil::MakeShape(xla::F32, {3, 4}));
```

Comment:
Perhaps a silly question, for the "op" in "ScalarOp", is it operator or operand? Because it's a scalar, so it looks like an operand. But the torch::lazy::Node is a "operation a Node can be associated to".

Reply:
It is a Node, check https://github.com/pytorch/xla/blob/227ac13d7102f4e30540df7ef50d09a02a985290/torch_xla/csrc/ops/ops.h#L17